rascal-lsp/src/main/rascal/demo/lang/pico/examples/ite.pico

```diff
@@ -0,0 +1,13 @@
+begin 
+declare 
+  input : natural,  
+  output : natural;
+  
+   input := 0;
+   output := 1;
+   if input then
+     output := 1
+   else
+     output := 2
+   fi
```

Comment:
Nit: Indentation is different vs. lines 3-4 (and also vs. `fac.pico`)

Reply:
:-) fixed. these examples were taken from flybytes where they were taken from the asf-library project in the meta-environment.